plugins/com.xored.q7.quality.mockups.swt/src/org/eclipse/swt/button/Button_Radio.java

```diff
@@ -37,7 +43,23 @@ public Control construct(Composite parent) {
 		
 		radioButtons[4] = new Button(composite, SWT.RADIO);
 		radioButtons[4].setToolTipText("Choice 4");
+
+		radioButtons[5] = new Button(composite, SWT.RADIO);
+		radioButtons[5].setText("Choice 5");
+
+		radioButtons[5].addSelectionListener(new SelectionAdapter() {
+			@Override
+			public void widgetSelected(SelectionEvent e) {
+                		selectionChange();
+            		}
+        	});
 		
 		return null;
 	}
+
+	private void selectionChange() {
+        boolean buttonSelected = radioButtons[5].getSelection();
+        text.setText(buttonSelected ? "selected" : "");

```

Comment:
A would recommend not use empty string here. "unselected" would be better visible on screenshots and easier to assert.

Reply:
fixed